R/tm_t_events.R

```diff
@@ -459,12 +454,15 @@ template_events <- function(dataname,
   y
 }
 
-#' Teal Module: Events by Term
+#' teal Module: Events by Term
+#'
+#' @seealso The [TLG Catalog](https://insightsengineering.github.io/tlg-catalog/stable/) where additional example
+#'   apps implementing this module can be found.
 #'
 #' @inheritParams module_arguments
 #' @inheritParams template_events
```

Comment:
Just a bit of rearranging + adding a description.

```suggestion
#' This module produces a table of events by term.
#'
#' @inheritParams module_arguments
#' @inheritParams template_events
#'
#' @seealso The [TLG Catalog](https://insightsengineering.github.io/tlg-catalog/stable/) where additional example
#'   apps implementing this module can be found.
```